static/src/js/components/ExternalLink/ExternalLink.jsx

```diff
@@ -21,6 +22,21 @@ export const ExternalLink = ({
     className
   ])
 
+  // If the link has a parent link this is needed to avoid DOM warnings while keeping styling
+  if (innerLink) {
```

Comment:
This is something I changed because there was a warning having a link element inside of a link element this keeps the styling consistent for the external links that we want but, avoids that DOM warning